database/metadata/databases/default/tables/public_crashes.yaml

```diff
@@ -17,15 +17,6 @@ object_relationships:
   - name: crashes_cri
     using:
       foreign_key_constraint_on: id
-  - name: crashes_list_view
```

Comment:
in the next PR where changes to the VZE are addressed i will need to recreate this view without references to the edits table